packages/core/README.md

```diff
@@ -0,0 +1,20 @@
+<div align="center">
+  <img src="https://raw.githubusercontent.com/sequelize/sequelize/main/logo.svg" width="100" alt="Sequelize logo" />
```

Comment:
```suggestion
  <img src="../../logo.svg" width="100" alt="Sequelize logo" />
```
Haven't checked but I think this would work. Reduces our dependency on external links

Reply:
Can't because that file won't be in the bundle that gets published on npm